analyzers/nodejs/nodejs.go

```diff
@@ -276,6 +276,15 @@ func recurseDeps(pkgMap map[pkg.ID]pkg.Package, p npm.Output) {
 		if ok {
 			continue
 		}
+
+        // Remove dev dependencies
+        if removeDev {
+            o := pkgMap[id]
+            if o.Qualifier == 2 {
```

Comment:
What is `o.Qualifier`?

Reply:
This refers to the `Qualifier` field in the `Package` struct: https://github.com/fossas/fossa-cli/blob/49419e96778b772845535654cb0f813bb922b6d9/pkg/package.go#L60